### ISSUE TITLE
Could not find gem 'polyamorous (= 2.3.1)'

### ISSUE THREAD

**Issue by Juneezee** [activerecord-hackery/ransack#1094]
When doing a bundle install, it will show 
```
Bundler could not find compatible versions for gem "polyamorous":
  In Gemfile:
    ransack was resolved to 2.3.1, which depends on
      polyamorous (= 2.3.1)

Could not find gem 'polyamorous (= 2.3.1)', which is required by gem 'ransack', in any of the sources.
```
I think this is caused by the latest version bump (commit 5e124dd1cca4ad3c38a0be7dab0f84f3fac8603f).

**Comment by seanfcarroll** [comment]
Thanks @Juneezee 😅 ... fixing it now.

**Comment by seanfcarroll** [comment]
Tests are in bad shape as the CI passed.

**Comment by seanfcarroll** [comment]
Please try it now @Juneezee

**Comment by Juneezee** [comment]
> Please try it now @Juneezee

Yes, everything is working now. Thanks for the quick fix.

**Comment by tvdeyen** [comment]
polyamorous 2.3.1 or 2.3.2 have not been released on rubygems yet #1098
